packages/lesswrong/lib/collections/surveyResponses/schema.ts

```diff
@@ -0,0 +1,62 @@
+import { foreignKeyField } from "@/lib/utils/schemaUtils";
+import { userOwns } from "@/lib/vulcan-users";
+
+const commonFields = ({nullable = false}: {
```

Comment:
```suggestion
const userEditableField = ({nullable = false}: {
```
nit: I think it would be slightly better to give this an informative name